static/app/views/organizationCreate/index.tsx

```diff
@@ -147,3 +161,7 @@ function OrganizationCreate() {
 }
 
 export default OrganizationCreate;
+
+const TermsWrapper = styled('div')<{hasDataConsent?: boolean}>`
+  margin-bottom: ${p => (p.hasDataConsent ? '0' : '16px')};
```

Comment:
Instead of this conditional margin, could you do a flex wrapper around both with `gap: ${space(2)};`

Reply:
because of how the form is set up, each checkbox by default has 16px bottom margin - my goal was to get rid of that for the terms one so that there isn't a gap if the data consent check is there. the flex wrapper i think would do the opposite of what i'm intending to (add the gap if the data consent check is there)

Reply:
Ah yeah you're right. One other thing you could try is a css selector, but that might be just as gnarly